backend/api/generic.go

```diff
@@ -95,6 +96,7 @@ func (g *GenericClient) Update(ctx context.Context, value corev2.Resource) error
 	if err := authorize(ctx, g.Auth, attrs); err != nil {
 		return err
 	}
+	setCreatedBy(ctx, value)
```

Comment:
It seems a bit odd to overwrite the `CreatedBy` field on update, but for now, it at least mirrors the `handler` package.

https://github.com/sensu/sensu-go/blob/master/backend/apid/handlers/update.go#L31-L36
```golang
	meta := resource.GetObjectMeta()
	if claims := jwt.GetClaimsFromContext(r.Context()); claims != nil {
		meta.CreatedBy = claims.StandardClaims.Subject
		resource.SetObjectMeta(meta)
	}
```